extensions/applicationinsights-clickanalytics-js/rollup.config.js

```diff
@@ -132,8 +136,10 @@ const nodeUmdRollupConfigFactory = (isProduction) => {
 updateDistEsmFiles(replaceValues, banner);
 
 export default [
+  browserRollupConfigFactory(true, version),
+  browserRollupConfigFactory(false, version),
+  browserRollupConfigFactory(true, verParts[0]),
+  browserRollupConfigFactory(false, verParts[0]),
```

Comment:
We should add at least the gbl format to the generation (same as the AISKU) otherwise if a site loads requirejs before the sdk the default module never initializes... We should also do the same for the debug plugin as well.